modules/extensions/src/data-filter/shader-module.ts

```diff
@@ -146,6 +146,12 @@ const inject = {
     #endif
   `,
 
+  'vs:#main-end': `
+    if (dataFilter_value == 0.0) {
+      gl_Position = vec4(-2, -2, -2, 1);
```

Comment:
`vec4(0.)` should suffice?

Reply:
The goal was to move all geometry outside of clip space, so anything outside of -1 -> 1 on any axis. I'm not a fan of the magic -2 values. Definitely open to suggestions. Any ideas?